README.md

```diff
@@ -1,2 +1,4 @@
 # review-inception
 This a repository made to understand how code review works.
+This reposoty has files copied from an existing repo : https://github.com/montsecruz9/Microverse-Trial.git
+This is for educational purposes only.
```

Comment:
I kindly suggest making a descriptive README file. 
This is because the README file is how you present your project to other people who could be interested in your work.